packages/web-ui/src/ds/molecules/Chat/Message/index.tsx

```diff
@@ -0,0 +1,43 @@
+import { MessageContent } from '@latitude-data/compiler'
+import { Badge, BadgeProps, Text } from '$ui/ds/atoms'
+import { TextColor } from '$ui/ds/tokens'
+import { cn } from '$ui/lib/utils'
+
+export type MessageProps = {
+  role: string
+  content: MessageContent[]
+  className?: string
+  badgeVariant?: BadgeProps['variant']
+  textColor?: TextColor
+}
+
+export function Message({
+  role,
+  content,
+  className,
```

Comment:
Don't use `className` as prop if possible. Everything should be a configuration.